src/backend/src/services/change-requests.services.ts

```diff
@@ -324,6 +326,18 @@ export default class ChangeRequestsService {
       }
     }
 
+    // send a reply to a CR's notifications of its updated status
+    try {
+      const relevantThreads = await prisma.message_Info.findMany({ where: { changeRequestId: foundCR.crId } });
+      relevantThreads?.forEach(async (thread) => {
```

Comment:
instead of this forEach, you should use map to create an array of promises, and resolve them with Promise.all (see how we use Promise.all elsewhere)